app/policies/roles/site/institution.json.erb

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "name": "Site <%=  %> Admin",
+    "policy": {
+      "statement": [
+        {
+          "resource": "site/<%= site_id %>",
+          "action": [
+            "site:read",
+            "site:update",
+            "site:delete"
+          ],
```

Comment:
Shouldn't the site admin have the same permissions as an institution admin, but scoped by site rather than institution?